ConsoleApp.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="RestSharp" Version="110.2.0" />
```

Comment:
One of the reasons why I went over time was because I was trying to utilize the built in http client handler: system.net.http
Doing this proved to be a major waste in time because I was struggling to use it effectively and quickly. Serialization also was an issue as I was initially using dictionaries for my deserialization when I got the request back. Due to the data being nested, I had to resort to creating custom objects. (I was programming like a JavaScript developer here trying to make on the fly objects as I expected in the API docs.)